src/app/GitExtUtils/GitUI/Interops/User32/GetWindowLong.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Runtime.InteropServices;
+
+namespace System;
+
+internal static partial class NativeMethods
+{
+    [DllImport("user32.dll")]
```

Comment:
```suggestion
    [DllImport("user32.dll", SetLastError = true, ExactSpelling = true)]
```
https://learn.microsoft.com/windows/win32/api/winuser/nf-winuser-getwindowlongptrw#return-value